Apps/WebClient/src/ClientApp/src/models/userPreference.ts

```diff
@@ -0,0 +1,8 @@
+export default interface UserPreference {
+    // Gets or sets the user preference id.
+    id?: string;
+    // Gets or sets the users directed identifier.
+    hdId: string;
+    // Gets or sets a value indicating if the user dismissed the note notification.
+    dismissedMyNotePopover: boolean;
```

Comment:
What do you think of tutorialPopover instead? Othwerwise we will need one for every single popover

Reply:
can you expand a bit further? do you mean using a custom tutorialPopover class instead of dismissedMyNotePopover: boolean?

Reply:
No I think a boolean is fine. The issue is that the name of the variable implies there can be more popover preferences such as dissmissedPrintPopover. Instead make it general for all popovers